public/app/features/dashboard-scene/scene/layout-rows/RowsLayoutManager.tsx

```diff
@@ -67,6 +83,10 @@ export class RowsLayoutManager extends SceneObjectBase<RowsLayoutManagerState> i
     return RowsLayoutManager.getDescriptor();
   }
 
+  public getSelectedObject() {
+    return sceneGraph.getAncestor(this, DashboardScene).state.editPane.state.selectedObject?.resolve();
```

Comment:
FYI: (can fix in main PR) we have getDashboardSceneFor which we usually use instead of sceneGraoh.getAncestor